package.json

```diff
@@ -55,12 +54,14 @@
     "eslint": "^5.16.0",
     "eslint-config-airbnb-base": "^13.2.0",
     "eslint-plugin-import": "^2.18.2",
+    "husky": "^3.1.0",
```

Comment:
what's this?

Reply:
it allows us to run scripts before we commit/push etc. I have set it up to run npm lint before pushing code. If linting fails the push fails. We could also run other scripts like test or ui-tests to make sure that we are not pushing broken code that could make it into master (I haven't included the test scripts though). It can be overridden by using `git push --no-verify` if you are working on something but want to push.

Reply:
Nice! I like that 👍 - let's discuss tomorrow as to what we think should be run for commit and push hooks